### ISSUE TITLE
Vulnerability with acorn

### ISSUE THREAD

**Issue by vishnutsivan** [angular/angular-cli#17899]
# 🐞 Bug report

Our security check tool is throwing a vulnerability issue with acorn 6.3.0 

### Description

@angular-devkit/build-angular@0.901.7 is dependent on less version 3.11.1 which has dependency with acorn@6.3.0.

## Possible Fix
We think we can solve this issue by updating less to version 3.11.3

**Comment by vishnutsivan** [comment]
[https://github.com/angular/angular-cli/pull/17898](url)

**Comment by alan-agius4** [comment]
Hi @vishnutsivan, can you please share the vulnerability report? less 3.11.1 doesn't depend on acorn.

https://unpkg.com/browse/less@3.11.1/package.json

**Comment by vishnutsivan** [comment]
@alan-agius4 in package.json of less 3.11.1 there is no dependency on acorn. But if we browse node_modules then we can find it's reference 
![image](https://user-images.githubusercontent.com/18438320/84259719-866b8b00-ab36-11ea-95c5-95d1d58b03a2.png)

**Comment by alan-agius4** [comment]
@vishnutsivan I see, the used to ship a node_modules directory as part of the package.

**Comment by vishnutsivan** [comment]
@alan-agius4 yes, and in the latest version of less ie 3.11.3 there are no such shipping of node_modules

**Comment by alan-agius4** [comment]
Note: this security notuice doesn't effect package manager audit tools such as NPM and Yarn.

@vishnutsivan, left you a couple of comments on the PR, thanks.
